### PR TITLE
Beta sticker

### PR DESCRIPTION
**Pull request by aapeliv**
Marketing people think we should make "betaness" more pronounced because we're light on features.

Screen Shot 2021-05-03 at 10 27 37 am

I do not know how to make pretty things.

**Frontend checklist**
- [ ] Formatted my code with `yarn format && yarn lint --fix`
- [ ] There are no warnings from `yarn lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [x] Added tests where relevant
- [ ] All tests pass
- [x] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/frontend/src/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -110,7 +111,7 @@ const useStyles = makeStyles((theme) => ({
 
 export default function Navigation() {
   const classes = useStyles();
-  const { logo } = useAuthStyles();
+  const { logo, beta } = useAuthStyles();
```
Comment (lucaslcode) [inline app/frontend/src/components/Navigation/Navigation.tsx]: Can you change this to `authClasses`

File: app/frontend/src/features/auth/useAuthStyles.ts [hunk 2]
```diff
@@ -123,6 +123,9 @@ const useAuthStyles = makeStyles((theme) => ({
     fontFamily: "'Mansalva', cursive",
     fontSize: "2rem",
   },
+  beta: {
+    marginLeft: theme.spacing(1),
+  },
```
Comment (lucaslcode) [inline app/frontend/src/features/auth/useAuthStyles.ts]: How about doing `position: relative; left: -3rem; transform: rotate(45deg);`? Too cliché?

**Review by lucaslcode — CHANGES_REQUESTED** [review]
This pushes the close button off to the side on mobile.
We could possibly just get rid of the close button, since clicking off the sidebar closes it.

**Comment by aapeliv** [comment]
Screen Shot 2021-05-10 at 6 49 09 pm

**Comment by aapeliv** [comment]
Screen Shot 2021-05-10 at 6 55 38 pm

How about this one?

**Comment by aapeliv** [comment]
Itsi was happy with the sticker on the logo.

**Review by lucaslcode — APPROVED** [review]
love it

**Pull request merged**